capstone/src/main/java/com/google/univiz/collegescorecarddata/CollegeScoreCard.java

```diff
@@ -0,0 +1,19 @@
+package com.google.univiz.collegescorecarddata;
```

Comment:
Again - I'm not sure that you want this API interface in a `collegescorecarddata` package. If the backing implementation changes - we may not use collegescorecard at all...

Reply:
changed!